tools/codegen/src/generator/languages/rust/mod.rs

```diff
@@ -40,9 +40,19 @@ impl super::LanguageGenerator for Generator {
     fn generate<W: io::Write>(writer: &mut W, ast: &ast::Ast) -> io::Result<()> {
         writeln!(writer, "// Generated by Molecule {}", VERSION)?;
         writeln!(writer)?;
+
+        #[cfg(feature = "gen-primitive-types")]
         let code = quote!(
             use molecule::prelude::*;
         );
+
```

Comment:
When `#[cfg(feature = "gen-primitive-types")]` , this happens when `molecule-std`'s `build.rs` is expanding primitive types code. So we only need `molecule`

Reply:
I only read code from your PR without doing compile, so maybe my opinion is not reasonable.

IMHO,

- if we let users control primitive types in users' crates, the feature should in generated code.

- if we control primitive types in `codegen`, I prefer to your old solution, this should be an option.